lucid_work/vis_1d.py

```diff
@@ -0,0 +1,105 @@
+# This file creates 1D visualizations for our model
+# Can create channel or neuron visualizwtions
+
+import numpy as np
+import tensorflow as tf
+import scipy.ndimage as nd
+import time
+import imageio
+
+import matplotlib
+import matplotlib.pyplot as plt
+
+import lucid.modelzoo.vision_models as models
+from lucid.misc.io import show
+import lucid.optvis.objectives as objectives
+import lucid.optvis.param as param
+import lucid.optvis.render as render
+import lucid.optvis.transform as transform
+
+from lucid.modelzoo.vision_base import Model
+
+class DLA(Model):
+    model_path = 'protobufs/full_model_8_13.pb'
+    image_shape = [1, 400]
+    image_value_range = [0,1]
+    input_name = 'x'
+
+PARAM_F = lambda: param.image(400, h=1, channels=1)
+THRESH = (256,)
+TFORMS = []
+
+
+LAYERS = { 'conv1': ['Conv2D', 100],
+           'conv1_relu': ['Relu', 100],
+           'pool1': ['MaxPool', 100],
+           'conv2': ['Conv2D_1', 96],
+           'conv2_relu': ['Relu_1', 96],
+           'pool2': ['MaxPool_1', 96],
+           'conv3': ['Conv2D_2', 96],
+           'conv3_relu': ['Relu_2', 96],
+           'pool3': ['MaxPool_2', 96]}
+
+def vis_channel(model, layer, channel_n):
```

Comment:
Can you add a brief doc string describing the data types of
`model`, `layer` and `channel_n` and what they are?